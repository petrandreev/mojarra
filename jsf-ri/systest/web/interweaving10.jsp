<%@page pageEncoding="UTF-8"%>
<%@taglib prefix="f" uri="http://java.sun.com/jsf/core"%>
<%@taglib prefix="h" uri="http://java.sun.com/jsf/html"%>


<f:view>
    <html>
        <body>
            <h:form id="form">
                <h:dataTable value="#{interweaving10.items}" var="item"
binding="#{interweaving10.table}">
                    <h:column>
                        <%-- 
                           Validates CASE 4 in UIComponentClassicTagBase when
                           using a component binding with a lifetime longer
                           than request
                        --%>
                         <h:outputText value="#{item}"/>
                        ciao                       
                    </h:column>
                </h:dataTable>               
            </h:form>
        </body>
    </html>
</f:view>

