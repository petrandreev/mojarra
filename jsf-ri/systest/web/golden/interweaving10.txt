





    <html>
        <body>
            <form id="form" name="form" method="post" action="/jsf-systest/faces/interweaving10.jsp;jsessionid=1d258701719f6d9621c6126a1b193" enctype="application/x-www-form-urlencoded">
<table>
<tbody>
<tr>
<td>Value
                        
                         
                        ciao                       
                    </td>
</tr>
</tbody>
</table>
<input type="hidden" name="javax.faces.ViewState" id="javax.faces.ViewState" value="j_id41:j_id42" /><input type="hidden" name="form" value="form" /></form>
        </body>
    </html>


